lib/ui/window/pointer_data_packet_converter.cc

```diff
@@ -84,6 +119,11 @@ void PointerDataPacketConverter::ConvertPointerData(
         auto iter = states_.find(pointer_data.device);
         FML_DCHECK(iter != states_.end());
         PointerState state = iter->second;
+        if (state.view_id != pointer_data.view_id) {
+          // Ignores remove event if the pointer was previously added to a
+          // different view.
+          break;
```

Comment:
Is this covered by the test you added? If not, could you update that test to cover this case or add a second test?

Reply:
It wasn't covered. I've added a new test.